BasicWebApp/AsyncServer.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Text;
+using BasicWebApp.Controllers;
+using BasicWebApp.Database;
+using BasicWebApp.DTO;
+using BasicWebApp.Exceptions;
+using BasicWebApp.Services;
+
+namespace BasicWebApp
+{
+    public class AsyncServer
+    {
+        private ControllerGreeting _controllerGreeting;
+        private ControllerPerson _controllerPerson;
+        private IController _controller;
+        private RequestParser _requestParser;
+
+        public AsyncServer()
+        {
+            List<Person> initialData = new List<Person>()
+            {
+                new Person(Constants.InitialPerson),
+            };
```

Comment:
I didn't have a particular reason for doing it this way. I think your suggestion is much cleaner. I have changed it to use AddPerson method instead of passing in a pre-populated list.